src/components/App.js

```diff
@@ -23,6 +22,17 @@ export default class App extends React.Component {
     error: PropTypes.object,
   };
 
+  getSnapshotBeforeUpdate() {
+    const { error } = this.props;
+
+    if (error) {
+      notification['error']({
+        message: 'Login Failed',
+        description: error.get('errorDescription'),
```

Comment:
What is this `errorDescription`? How does it get into `state.auth.session.error`?